### ISSUE TITLE
Game lags when there are assigned Quickslots

### ISSUE THREAD

**Issue by GoogleCodeExporter** [alistairreilly/andors-trail#266]
```
Reported here:
http://andorstrail.com/viewtopic.php?p=6874#p6874

When you unassign the Quickslots and disable the view of them, there is a big 
performance boost.

There is no performance problem with the d-pad.
Perhaps the quickslots should be implemented in a similar way.
```

Original issue reported on code.google.com by `SamuelPl...@gmail.com` on 2 Nov 2011 at 11:45

**Comment by GoogleCodeExporter** [comment]
```
Perhaps we should release this as a bugfix release. (without new content)
Many complained about the slowness.
```

Original comment by `SamuelPl...@gmail.com` on 2 Nov 2011 at 12:03

**Comment by GoogleCodeExporter** [comment]
```
Confirmed
```

Original comment by `sdeva...@gmail.com` on 2 Nov 2011 at 5:19
- Changed state: **Accepted**
- Added labels: **Priority-High**

**Comment by GoogleCodeExporter** [comment]
```
Has oskar committed a fix for this problem? This is one of the first things I 
was going to look into fixing. Have a pretty good idea on an easy fix, but need 
to get a little deeper into the code before I can say for sure
```

Original comment by `taxijef...@gmail.com` on 18 Jan 2012 at 12:42

**Comment by GoogleCodeExporter** [comment]
```
AFAIK he hasn't looked into this yet. You are welcome to contribute.
```

Original comment by `SamuelPl...@gmail.com` on 18 Jan 2012 at 4:50

**Comment by GoogleCodeExporter** [comment]
```
Hello taxijeffoc,

this was partially fixed in r201, and even more refined in r203, thus 
preventing future caching problems of the tile images.

Credits should go to forum user "chief" for finding it.
```

Original comment by `oskar.wi...@gmail.com` on 18 Jan 2012 at 7:10
- Changed state: **Ready**
- Added labels: **Milestone-0.6.11**

**Comment by GoogleCodeExporter** [comment]
```
Released in v0.6.11. Closing issue.
```

Original comment by `oskar.wi...@gmail.com` on 9 Jun 2012 at 9:31
- Changed state: **Fixed**
